modules/openapi-generator/src/test/java/org/openapitools/codegen/java/spring/SpringCodegenTest.java

```diff
@@ -987,6 +987,19 @@ public void oneOf_5381() throws IOException {
         assertFileContains(Paths.get(outputPath + "/src/main/java/org/openapitools/model/FooRefOrValue.java"), "public interface FooRefOrValue");
     }
 
+    @Test
+    public void oneOf_11932() throws Exception {
+        final SpringCodegen codegen = new SpringCodegen();
+        codegen.setUseOneOfInterfaces(true);
+
+        final Map<String, File> files = generateFiles(codegen, "src/test/resources/3_0/issue_11932-oneOf.yaml");
+
+        final File fruitBaseClass = files.get("Fruit.java");
+        final String line = "public Fruit putBodyItem(String key,  bodyItem)";
```

Comment:
Does the resulting Java compile? This looks like a method declaration with no type specified, but maybe I'm lacking context

Reply:
Yes, it compiles just fine. In order to see the full method definition, you will need to click the Test class link https://github.com/OpenAPITools/openapi-generator/pull/11987/files/60f8c49faf02a07b3021e91839928b2c20d5be99#diff-4ce5fb739894dacffe829d7ec59dea6513fb16d9e2cfd2ab495098302037382f

Reply:
If I generate the code with your test yaml and changes applied running `java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g spring -i modules/openapi-generator/src/test/resources/3_0/issue_11932-oneOf.yaml -o /tmp/spring/`

I do not get a `Fruit.java` class that has a method `putBodyItem` - [see here](https://gist.github.com/welshm/13f023560ec5bd234451d071edb4e064)

Reply:
I think this is expected, the assertion is `assertFileNotContains`.

Reply:
My mistake - should read the assertion more closely